CMakeLists.txt

```diff
@@ -463,37 +463,69 @@ ELSE (OPENSSL_FOUND)
     SET_TARGET_PROPERTIES(libh2o-evloop PROPERTIES EXCLUDE_FROM_ALL 1)
 ENDIF (OPENSSL_FOUND)
 
-ADD_CUSTOM_TARGET(lib-examples DEPENDS examples-http1client examples-simple examples-socket-client)
+ADD_CUSTOM_TARGET(lib-examples DEPENDS examples-http1client examples-simple examples-socket-client examples-latency-optimization)
 IF (WSLAY_FOUND)
     ADD_DEPENDENCIES(lib-examples examples-websocket)
 ENDIF (WSLAY_FOUND)
-
 ADD_EXECUTABLE(examples-http1client examples/libh2o/http1client.c)
+ADD_EXECUTABLE(examples-socket-client examples/libh2o/socket-client.c)
+ADD_EXECUTABLE(examples-simple examples/libh2o/simple.c)
+ADD_EXECUTABLE(examples-websocket lib/websocket.c examples/libh2o/websocket.c)
+ADD_EXECUTABLE(examples-latency-optimization examples/libh2o/latency-optimization.c)
+
+ADD_CUSTOM_TARGET(lib-examples-evloop DEPENDS examples-http1client-evloop examples-simple-evloop examples-socket-client-evloop)
```

Comment:
Please add `lib-examples-evloop` as an dependency of `check`, so that we can detect build errors of the examples when `make check` is being run.

Reply:
fixed